src/rc/start-stop-daemon.c

```diff
@@ -1211,7 +1222,8 @@ start_stop_daemon(int argc, char **argv)
 			if ((strncmp(env->value, "RC_", 3) == 0 &&
 				strncmp(env->value, "RC_SERVICE=", 10) != 0 &&
 				strncmp(env->value, "RC_SVCNAME=", 10) != 0) ||
-			    strncmp(env->value, "SSD_NICELEVEL=", 14) == 0)
+				strncmp(env->value, "SSD_NICELEVEL=", 14) == 0 ||
```

Comment:
This indentation makes things confusing, because it suggests the logic depends on the strncmp of RC_